test/CI-blackbox/java/renderer.js

```diff
@@ -0,0 +1,22 @@
+
+const {JavaGenerator} = require('../../../lib');
+const fs = require('fs');
```

Comment:
you can do this, just a hint
```suggestion
const fsp = require('fs').promises;
```